sale_commission_areamanager/views/res_partner_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_form_area_manager" model="ir.ui.view">
+        <field name="name">res.partner.form.area.manager</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="sale_commission.view_partner_form_agent" />
+        <field name="priority" eval="18"/>
+        <field name="arch" type="xml">
+                <page name="agent_information" position="replace">
+                    <page name="agent_information" string="Agent information" attrs="{'invisible': [('agent', '=', False)]}">
+                        <group>
+                            <group col="2">
+                                <field name="agent_type"/>
+                                <field name="commission" attrs="{'required': [('agent', '=', True)]}"/>
+                                <field name="settlement"/>
+                            </group>
+                            <group col="2">
+                                <field name="area_manager"/>
+                                <field name="area_manager_id" attrs="{'invisible': [('area_manager', '=', False)]}"/>
```

Comment:
here new records can be created, so I would add default values for agent (True) and area_manager (True) plus the correct form view to open